lib/MIME/Types.rakumod

```diff
@@ -5,7 +5,7 @@ unit class MIME::Types;
 has %.exts;
 has %.types;
 
-method new (Stringy $mtfile) {
+method new (Str() $mtfile = %?RESOURCES<mime.types>) {
   if $mtfile.IO !~~ :f {
```

Comment:
I know this is legacy code, but as long as we're improving this, wouldn't it be better to throw a more specific, typed exception? There's probably one you can take off-the-shelf.